url/urltestdata.txt

```diff
@@ -1,402 +0,0 @@
-# FORMAT NOT DOCUMENTED YET (parser is urltestparser.js)
-
-# Based on http://trac.webkit.org/browser/trunk/LayoutTests/fast/url/script-tests/segments.js
```

Comment:
That or maybe in a README. They have been changed a bit too so the reference is no longer that valuable.